doc/source/whatsnew/v1.1.1.rst

```diff
@@ -51,6 +51,10 @@ Categorical
 -
 -
 
+**Groupby/resample/rolling**
+
+- Bug in :class:`pandas.core.groupby.RollingGroupby` where passing ``closed`` with column selection would raise a ``ValueError`` (:issue:`35549`)
```

Comment:
Wasn't a regression from 1.1.0, but thought to include it in 1.1.1 since I'm fixing other `groupby.rolling` bugs.